add-game/index.md

```diff
@@ -0,0 +1,83 @@
+## How do I add my game to Game Jolt?
+
+First of all, you need to have a developer account. If you don't yet have one, just [go here](http://gamejolt.com/dashboard/become-developer/). 
+
+1. If you don't have a Game Jolt account at all, [sign up](http://gamejolt.com/auth/sign_up/) for one, and be sure to choose "Developer" when the time comes. 
+2. Developer account all set up? Good. click/tap "Dashboard" on the black nav bar. Welcome to your [developer dashboard](http://gamejolt.com/dashboard/)! Now hit the big "Add a Game" button. Go ahead, do it. 
+
+### Step 1: Add game information
+
+You'll need to fill in some basic information about your game in order to proceed.
+
+#### Title
+
+Surely your game has a name. Put it here.
+
+- A note on mature language: If your game's title contains the "F-word" or a swear word of that magnitude, please replace one or more of its letters with asterisks or similar characters. This will keep browsing friendly for people who do not wish to view mature content. This only applies to titles and thumbnails because they show up in lists and search results; once someone visits a game page, there is a maturity rating on display and one can make their own decision about whether to read the description and play the game. 
+
+#### Website 
+
+If you wish to link to a personal website about the game or about your games in general, this is the place to do it.
+
+#### Category
+
+You have to choose a category for your game. The category you pick is entirely up to you. Categories are meant to be helpful, not restrictive. They are signposts to aid gamers in browsing the site and finding the games they want to play. You might want to browse the category listings yourself to see what kinds of games have been assigned to them in the past.
```

Comment:
The last sentence "You might want to browse the category listings yourself to see what kinds of games have been assigned to them in the past." I don't think you can actually do this on WTTF so we should update.